### PR TITLE
ServiceBus: Added validator for rights

### PR DESCRIPTION
**Pull request by v-Ajnava**
---
Fixes: #5824 

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/servicebus/_validators.py [hunk 1]
```diff
@@ -116,3 +116,9 @@ def validate_subnet(cmd, namespace):
     else:
         raise CLIError('incorrect usage: [--subnet ID | --subnet NAME --vnet-name NAME]')
     delattr(namespace, 'vnet_name')
+
+
+def validate_rights(namespace):
+    if 'Manage' in namespace.rights:
+        if 'Listen' not in namespace.rights or 'Send' not in namespace.rights:
+            raise CLIError('Error : Assigning \'Manage\' to --rights requires \'Listen\' and \'Send\' to be included with. e.g. --rights Manage Send Listen')
```
Comment (tjprescott) [inline src/azure-cli/azure/cli/command_modules/servicebus/_validators.py]: This seems like it should be a service-side error.

Comment (v-Ajnava) [reply src/azure-cli/azure/cli/command_modules/servicebus/_validators.py]: @axisc, can you please share your view on this?

**Pull request merged**